lace_fake/snapshot_admin.py

```diff
@@ -24,14 +24,21 @@ def __init__(self,snap_json,scales_tau=None,kF_Mpc=None,post_dir=None):
             print('use',post_dir,'as post-processing directory')
             self.data['post_dir']=post_dir
             self.data['raw_dir']=None
-            self.data['skewers_dir']=post_dir+'/skewers/'
         else:
             if 'post_dir' in self.data:
                 print('no post_dir provided, will use',self.data['post_dir'])
-                self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
+                # self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
```

Comment:
We should think of all possible use cases here. Old files from Chris (that we want to use for the P3D emulator) do not have post_dir in the json files. However, I guess for those we should pass post_dir explicitly, so it is probably fine. I guess we can merge and come back to fix this if something is crashing.

Reply:
Note that the original function was:

```
if post_dir:
            print('use',post_dir,'as post-processing directory')
            self.data['post_dir']=post_dir
            self.data['raw_dir']=None
            self.data['skewers_dir']=post_dir+'/skewers/'
        else:
            if 'post_dir' in self.data:
                print('no post_dir provided, will use',self.data['post_dir'])
                self.data['skewers_dir']=self.data['post_dir']+'/skewers/'
            else:
                raise ValueError('must provide post_dir when using old files')
```

so I did not change anything there. Anyway, let's figure out if crashes

Reply:
You are right, you are defining skewers_dir later on now. I wrote my comment too early.